lib/pxenforcer.js

```diff
@@ -73,6 +83,50 @@ class PxEnforcer {
         }
     }
 
+    isTelemetryCommand(req) {
+
+        const headerVal = req.headers[this._config.TELEMETRY_COMMAND_HEADER];
+
+        if(!headerVal) {
+            return false;
+        }
+
+        this.logger.debug('Received command to send enforcer telemetry');
+
+        //base 64 decode
+        const decodedString = Buffer.from(headerVal, 'base64').toString();
```

Comment:
Better adding `.toString('utf8')` just in case there's some encoding issue.

Reply:
@ilaipx `utf8` is the [default](https://nodejs.org/api/buffer.html#buffer_buf_tostring_encoding_start_end)